make/autoconf/help.m4

```diff
@@ -257,115 +257,117 @@ AC_DEFUN_ONCE([HELP_PRINT_SUMMARY_AND_WARNINGS],
 [
   # Finally output some useful information to the user
 
-  printf "\n"
-  printf "====================================================\n"
+  $ECHO ""
+  $ECHO "===================================================="
   if test "x$no_create" != "xyes"; then
     if test "x$IS_RECONFIGURE" != "xyes"; then
-      printf "A new configuration has been successfully created in\n%s\n" "$OUTPUTDIR"
+      $ECHO "A new configuration has been successfully created in"
+      $ECHO "$OUTPUTDIR"
     else
-      printf "The existing configuration has been successfully updated in\n%s\n" "$OUTPUTDIR"
+      $ECHO "The existing configuration has been successfully updated in"
+      $ECHO "$OUTPUTDIR"
     fi
   else
     if test "x$IS_RECONFIGURE" != "xyes"; then
-      printf "A configuration has been successfully checked but not created\n"
+      $ECHO "A configuration has been successfully checked but not created"
     else
-      printf "The existing configuration has been successfully checked in\n%s\n" "$OUTPUTDIR"
+      $ECHO "The existing configuration has been successfully checked in"
+      $ECHO "$OUTPUTDIR"
     fi
   fi
   if test "x$CONFIGURE_COMMAND_LINE" != x; then
-    printf "using configure arguments '$CONFIGURE_COMMAND_LINE'.\n"
+    $ECHO "using configure arguments '$CONFIGURE_COMMAND_LINE'."
   else
-    printf "using default settings.\n"
+    $ECHO "using default settings."
   fi
 
   if test "x$REAL_CONFIGURE_COMMAND_EXEC_FULL" != x; then
-    printf "\n"
-    printf "The original configure invocation was '$REAL_CONFIGURE_COMMAND_EXEC_SHORT $REAL_CONFIGURE_COMMAND_LINE'.\n"
+    $ECHO ""
+    $ECHO "The original configure invocation was '$REAL_CONFIGURE_COMMAND_EXEC_SHORT $REAL_CONFIGURE_COMMAND_LINE'."
   fi
 
-  printf "\n"
-  printf "Configuration summary:\n"
-  printf "* Name:           $CONF_NAME\n"
-  printf "* Debug level:    $DEBUG_LEVEL\n"
-  printf "* HS debug level: $HOTSPOT_DEBUG_LEVEL\n"
-  printf "* JVM variants:   $JVM_VARIANTS\n"
-  printf "* JVM features:   "
+  $ECHO ""
+  $ECHO "Configuration summary:"
+  $ECHO "* Name:           $CONF_NAME"
+  $ECHO "* Debug level:    $DEBUG_LEVEL"
+  $ECHO "* HS debug level: $HOTSPOT_DEBUG_LEVEL"
+  $ECHO "* JVM variants:   $JVM_VARIANTS"
+  $ECHO "* JVM features:   "
```

Comment:
I believe this will now add a newline where there was none earlier, so the actual features will show up on a new line?

Reply:
Good catch; I'll fix that.

Reply:
Again `echo -n`?